.github/pull_request_template.md

```diff
@@ -0,0 +1,6 @@
+## Things done
+
+<!-- Please update the relevant checkboxes to mark that they have been done -->
+
+- [ ] Updated the documentation for this component, where appropriate.
+- [ ] For major changes, updated the `CHANGES.md` file.
```

Comment:
I do the changes during release time, it works better that having people do it as they go in my opinion.